Issue 3039

anisa-hawes opened the pull request:
I'm moving former (non-renewing) IPPs to the [Additional Supporters section](https://programminghistorian.org/en/supporters#additional-supporters) of our /en/supporters, /es/colaboradores, /fr/nos-soutiens, /pt/apoiadores pages.

Closes #3039 

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Add the appropriate "Label"
- [x] If this PR closes an Issue, add the phrase `Closes #ISSUENUMBER` to your summary above
- [x] Ensure the status checks pass: if you have difficulty fixing build errors, please contact our Publishing Assistant @anisa-hawes 
- [x] Check the Netlify Preview: navigate to netlify/ph-preview/deploy-preview and click 'details' (at right)
- [ ] Assign at least one individual or team to "Reviewers"
  - ~~[ ] if the text needs to be translated, please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines), then assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing editor in your PR.~~

anisa-hawes committed:
Update supporters.md

Former (non-renewing) IPPs > Additional Supporters
Make typography consistent, lines 58-76

anisa-hawes committed:
Update colaboradores.md

Former (non-renewing) IPPs > Additional Supporters
Make typography consistent, lines 61-79

anisa-hawes committed:
Update nos-soutiens.md

Former (non-renewing) IPPs > Additional Supporters
Make typography consistent, lines 59-77

anisa-hawes committed:
Update apoiadores.md

Former (non-renewing) IPPs > Additional Supporters
Make typography consistent, lines 60-80

anisa-hawes committed:
Update supporters.md

Update contact, line 110.

anisa-hawes committed:
Update colaboradores.md

Update contact, line 113.

anisa-hawes committed:
Update nos-soutiens.md

Update contact, line 111.

anisa-hawes committed:
Update apoiadores.md

Update contact, line 114

anisa-hawes committed:
Update ipp.md

Update contact at lines 35, 58 and 80.

anisa-hawes committed:
Update pia.md

Update contact at lines 36, 60 and 87.

anisa-hawes committed:
Update pi.md

Update contact at lines 38, 63 and 84.

anisa-hawes committed:
Update ppi.md

Update contact at lines 38, 53 and 71.

Commit:
Merge branch 'gh-pages' into Issue-3039

anisa-hawes committed:
Update ppi.md

Add alert-info box to make consistent across /en/ipp, /es/pia, /fr/pi, /pt/ppi

anisa-hawes committed:
Update ppi.md

Add html <li> and <div> tags, lines 63-70

anisa-hawes committed:
Update ppi.md

Replace Markdown link syntax with html at line 61.

anisa-hawes committed:
Update supporters.md

- Create Alumni Partners sub-section
- Move (non-renewing) IPPs to Alumni Partners
- Sub-divide Additional Supporters into active and past

anisa-hawes committed:
Update colaboradores.md

- Create Alumni Partners sub-section
- Move (non-renewing) IPPs to Alumni Partners
- Sub-divide Additional Supporters into active and past

anisa-hawes committed:
Update supporters.md

Minor edits, lines 69 and 73

anisa-hawes committed:
Update nos-soutiens.md

- Create Alumni Partners sub-section
- Move (non-renewing) IPPs to Alumni Partners
- Sub-divide Additional Supporters into active and past

anisa-hawes committed:
Update apoiadores.md

- Create Alumni Partners sub-section
- Move (non-renewing) IPPs to Alumni Partners
- Sub-divide Additional Supporters into active and past

anisa-hawes committed:
Update ipp.md

Update month when we reset partnership rates.

anisa-hawes committed:
Update pia.md

Update month when we reset partnership rates.

anisa-hawes committed:
Update pi.md

Update month when we reset partnership rates.

anisa-hawes committed:
Update ppi.md

Update month when we reset partnership rates.

anisa-hawes commented:
Hello @charlottejmc,

When you have time, may I ask for your help to check my French on the following page?:
[ph-preview.netlify.app/fr/nos-soutiens](https://deploy-preview-3040--ph-preview.netlify.app/fr/nos-soutiens).

What I've done is: 

- Added a new heading Alumni Partners (_Partenaires des anciens élèves_) 
- Added a description to mean roughly Former contributors to our Institutional Partner Programme (_Les institutions suivantes ont participé de notre programme de Partenariat institutionnel_). Does this sound right?
- Added two sub-headings under Additional Supporters (_Soutien additionnel_) which I'd like to read the equivalent of Active and Past. Would _Actifs_ and _Passé_ seem right?

Thank you 🙂

charlottejmc commented:
Hi @anisa-hawes,

I have had a look and here are my thoughts:

- _Partenaires des anciens élèves_ sounds more like _Partners of alumni_. I am thinking _Anciens élèves partenaires_ instead – but this is a tricky phrase to translate.
- This is nearly right: _Les institutions suivantes ont participé **à** notre programme de Partenariat institutionnel_ is the only correction I can see!
- I think this could work as it is, although _Passé_ would just need to be in the plural form: _Passés_. However, I might suggest something along the lines of _En cours_ and _Révolus_, which could sound more natural? Again, I feel like these are tricky phrases to translate directly.

I hope this helps!

anisa-hawes committed:
Update nos-soutiens.md

Adjustments to translation at lines 55, 56, 69 and 72.

spapastamkou commented:
hi, if I may, for the term "alumni partners", simply "anciens partenaires" ou "partenariats révolus" (as Charlotte proposes) can work here. (as élèves refers to students, and I believe is embarked from the auto translation of the initial alumni term)

anisa-hawes committed:
Update nos-soutiens.md

Adjust translation, line 55

Commit:
Merge branch 'gh-pages' into Issue-3039

rivaquiroga committed:
ES updates

anisa-hawes commented:
Hello @DanielAlvesLABDH,

I hope you're well. This PR represents some adjustments to the headings and sub-headings on our Supporters pages. Our aim is to create meaningful sub-headings within our lists of IPP, so we make clear who our active supporters are while also acknowledging institutions that have supported us in the past. May I ask for your help to adjust my attempted Portuguese translations?

- I added a new heading Alumni Partners (Parceiros ex-alunos? or Antigos parceiros?)
- I added a description to mean roughly Former contributors to our Institutional Partner Programme (Antigos contribuintes para o nosso Programa de Parceria Institucional)
- I added two sub-headings under Additional Supporters (Apoiadores adicionais) which I'd like to read the equivalent of Active and Past. Would Ativo and Anterior be appropriate?

You can [review the detail of my edits here](https://github.com/programminghistorian/jekyll/pull/3040/files). 

With thanks! Anisa

DanielAlvesLABDH commented:
Hi @anisa-hawes, will look at this tomorrow morning. Best

DanielAlvesLABDH commented on pt/apoiadores.md:
```diff
@@ -59,24 +54,36 @@ Contribuintes para o nosso [Programa de Parceria Institucional](/pt/ppi).
 Aceitamos consultas de possíveis contribuintes para o [Programa de Parceria Institucional](/pt/ppi).
 
-## Apoiadores adicionais
+## Parceiros ex-alunos
```
Anteriores parceiros

DanielAlvesLABDH commented on pt/apoiadores.md:
```diff
@@ -59,24 +54,36 @@ Contribuintes para o nosso [Programa de Parceria Institucional](/pt/ppi).
+- [RStudio/Posit](https://posit.co/), Estados Unidos [2022].
 
+
+## Apoiadores adicionais
```
Parceiros adicionais

DanielAlvesLABDH reviewed (changes requested):
Have made some suggestions of change

anisa-hawes committed:
Update apoiadores.md

Adjust translations lines 57 and 69

anisa-hawes commented:
Hello @spapastamkou and @rivaquiroga 

This PR is ready, with each of the translations checked by native speakers. 

The only further change I've made is to update the information about which month we reset partnership rates. As discussed with @drjwbaker and @spapastamkou, going forwards will be reviewing our rates mid-year for reset in October (rather than January) to align with ProgHist Ltd's financial year. 

Thank you.
